src/sentry/identity/github/provider.py

```diff
@@ -38,7 +40,10 @@ def get_oauth_client_secret(self):
 
     def build_identity(self, data):
         data = data["data"]
-        user = get_user_info(data["access_token"])
+        access_token = data.get("access_token")
+        if not access_token:
+            return Response(status=403)
```

Comment:
The usage of `build_identity()` is
```py
identity = super().build_identity(data)
```
so I think you'd have to do something like
```py
raise HTTPError(code=403)
```

Reply:
Bleh sorry that's not valid usage of `HTTPError`. You could try
```py
from django.core.exceptions import PermissionDenied
...
raise PermissionDenied()
```